### PR TITLE
Physical object delete unlinked task, refs #12630

### PR DESCRIPTION
**Pull request by mcantelon**
EDIT: Issue number in branch is wrong, but one in commit is correct.

Added task to delete any physical objects that aren't linked to an
information object.

### CODE REVIEW

**Comment by mcantelon** [comment]
Will be adding this development to another branch.

File: lib/task/physicalobject/physicalObjectDeleteUnlinkedTask.class.php [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+      }
+    }
+
+    // Disable search index
+    QubitSearch::disable();
+
+    // Display initial count of physical objects
+    $physicalObjectsCountBefore = $this->getPhysicalObjectCount();
+
+    $this->log();
```
Comment (jraddaoui) [inline lib/task/physicalobject/physicalObjectDeleteUnlinkedTask.class.php]: Is this to add an empty line in the output? Maybe adding `\n` to the following log looks a little better, but if it's not completely necessary I'd remove the empty line from the logs.

File: lib/task/physicalobject/physicalObjectDeleteUnlinkedTask.class.php [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+    {
+      $physicalObjectsCountAfter = $this->getPhysicalObjectCount();
+    }
+    else
+    {
+      // Simulate results during dry run
+      $physicalObjectsCountAfter = $physicalObjectsCountBefore - count($toDelete);
+    }
+
+    $this->log();
```
Comment (jraddaoui) [inline lib/task/physicalobject/physicalObjectDeleteUnlinkedTask.class.php:130]: Same here ;)

**Review by jraddaoui — APPROVED** [review]
Nicely done!

**Pull request merged**